### PR TITLE
Add tests for whitespace alt values in srcset within html-aam

### PR DESCRIPTION
**Pull request by giacomo-petri**
Expands the tests for the img element to verify if the correct role is being exposed for the img element with `alt=" "`.

Related to: https://github.com/w3c/html-aam/issues/605
Closes: https://github.com/web-platform-tests/interop-accessibility/issues/238

### CODE REVIEW

File: html-aam/img-src-srcset-roles.tentative.html
Comment (adampage) [inline html-aam/img-src-srcset-roles.tentative.html:22]:
```suggestion
<div hidden id=labelledby>x</div>
<div hidden id="empty"></div>
<div hidden id="space"> </div>
```

Heya @giacomo-petri, you didn’t introduce this, but [the new subtests on line 120 and 121](https://github.com/web-platform-tests/wpt/pull/60401/changes#diff-423194e5b855b345bbba8736bf0f0a1ddff096e4759f47c9447f3d2c5f793396R120) (which imitates the [pre-existing subtests on line 80 and 82](https://github.com/web-platform-tests/wpt/pull/60401/changes#diff-423194e5b855b345bbba8736bf0f0a1ddff096e4759f47c9447f3d2c5f793396R80)) are expecting names from IDREFs `empty` and `space` respectively.

I’m guessing there were meant to be additional `div`s up here to serve that purpose? 🤔

File: html-aam/img-src-srcset-roles.tentative.html [hunk 1]
```diff
@@ -113,6 +113,20 @@ <h2>Presentation / None / Generic role</h2>
...
+<img data-testname="el-img-whitespace-alt-title" class="ex-generic" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" alt=" " title="x">
+<img data-testname="el-img-whitespace-alt-title-empty" class="ex-generic" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" alt=" " title="">
+<img data-testname="el-img-whitespace-alt-title-whitespace" class="ex-generic" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" alt=" " title=" ">
+<img data-testname="el-img-whitespace-alt-no-src-or-srcset" class="ex-generic" alt=" ">
+<img data-testname="el-img-whitespace-alt-no-src-srcset-title-empty" class="ex-generic" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" alt=" " title="">
+<img data-testname="el-img-whitespace-alt-no-src-srcset-title-whitespace" class="ex-generic" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" alt=" " title=" ">
```
Comment (adampage) [inline html-aam/img-src-srcset-roles.tentative.html]:
```suggestion
<img data-testname="el-img-whitespace-alt-no-src-srcset-title-empty" class="ex-generic" alt=" " title="">
<img data-testname="el-img-whitespace-alt-no-src-srcset-title-whitespace" class="ex-generic" alt=" " title=" ">
```

These two subtests have unique `data-testname` values, but their testing of `title=""` and `title=" "` seems functionally redundant with [the subtests on lines 123 and 124](https://github.com/web-platform-tests/wpt/pull/60401/changes#diff-423194e5b855b345bbba8736bf0f0a1ddff096e4759f47c9447f3d2c5f793396R123-R124)?

Are they meant to have no `srcset`?

**Review by adampage — CHANGES_REQUESTED** [review]

**Comment by giacomo-petri** [comment]
Thanks @adampage, suggestions applied

**Review by spectranaut — APPROVED** [review]
Awesome, thanks!

**Review by adampage — APPROVED** [review]
Ship it. 🚀
